database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/LoginView.swift

```diff
@@ -19,19 +19,103 @@ import Firebase
 
 struct LoginView: View {
   @StateObject var user = UserViewModel()
-  var screenWidth = UIScreen.main.bounds.width
-  var screenHeight = UIScreen.main.bounds.height
+  #if os(iOS)
+    let screenWidth = UIScreen.main.bounds.width
+    let screenHeight = UIScreen.main.bounds.height
+  #endif
 
-  var body: some View {
-    NavigationView {
+  #if os(iOS)
```

Comment:
After talking to Maksym about the `#if` conditionals, I realize it will be the best to reuse as much code as possible. So instead I will try to use the same body for different platforms.